scripts/west_commands/runners/core.py

```diff
@@ -649,6 +660,15 @@ def dev_id_help(cls) -> str:
                   target when multiple ones are available or
                   connected.'''
 
+    @classmethod
+    def extload_help(cls) -> str:
+        ''' Get the ArgParse help text for the --extload option.'''
+        return '''External loader to be used by stm32cubeprogrammer
+                  to program the targeted external memory.
+                  This must be given one time; the resulting argument
+                  will be given to the tool in the order they appear
+                  on the command line preceded by -el.'''
```

Comment:
This part is useless as well, only one argument is required and needed.

Reply:
ok